csrankings.csv

```diff
@@ -4026,7 +4026,8 @@ Di Yuan,Uppsala University,https://katalog.uu.se/empinfo/?id=N15-9,NOSCHOLARPAGE
 Dia Ali,University of Southern Mississippi,https://www.usm.edu/computing/faculty/dia-ali,NOSCHOLARPAGE
 Dian Zhang,Shenzhen University,http://nhpcc.szu.edu.cn/zhangdian,tiE0IDMAAAAJ
 Diana F. Adamatti,FURG Federal University of Rio Grande,http://www.diana.c3.furg.br,anyT2ZkAAAAJ
-Diana Francisca Adamatti,FURG Federal University of Rio Grande,http://www.diana.c3.furg.br,anyT2ZkAAAAJ
+Diana Francisca Adamatti,FURG Federal University of Rio Grande,http://www.diana.c3.furg.br,anyT2ZkAAAAJ
+Diana Franklin,University of Chicago,http://people.cs.uchicago.edu/,~dmfranklin/,gG5PRvgAAAAJ

```

Comment:
There's a spurious comma in the URL (before the `~` character).

Reply:
Sorry!  Done.